core/common/src/TextureMapping.ts

```diff
@@ -74,17 +78,17 @@ export namespace TextureMapping {
     Parametric = 0,
     ElevationDrape = 1,
     Planar = 2,
-    /** @internal */
+    /** Currently unsupported. */
```

Comment:
if theyre not supported any reason we just dont remove them?

Reply:
They exist. Connectors preserve them. The renderer just currently does not handle them.